.github/workflows/groovy-tests.yml

```diff
@@ -19,3 +19,19 @@ jobs:
         uses: codecov/codecov-action@v3.1.0
         with:
           files: ./build/reports/jacoco/test/jacocoTestReport.xml
+
+  test-windows:
+    runs-on: windows-latest
+    steps:
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - uses: actions/checkout@v2
+      - name: Run Tests
+        run: |
+          ./gradlew test --info
+      - name: Upload Coverage Report
+        uses: codecov/codecov-action@v3.1.0
+        with:
+          files: ./build/reports/jacoco/test/jacocoTestReport.xml
```

Comment:
Instead of adding this, you can add a matrix of both systems. Example: https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/precommit.yml#L7-L10
Another example: https://github.com/opensearch-project/opensearch-build/blob/main/.github/workflows/python-tests.yml#L10-L16

Reply:
Here is the workflow run for PR, https://github.com/rblcoder/opensearch-build-libraries/pull/1/checks
The matrix looks great, I will try that.

Reply:
I have added a matrix, here is a workflow run https://github.com/rblcoder/opensearch-build-libraries/actions/runs/6559531559/job/17815315258